### ISSUE TITLE
SA1111 message for attribute brackets is confusing

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1557]
Currently SA1111 is reported for `]` character in the following code:

```csharp
[
Obsolete()
]
```

The message provided for this diagnostic is:

> Closing parenthesis must be on line of last parameter

The code fix for the diagnostic changes the code to the following:

```csharp
[
Obsolete()]
```

:memo: Note that SA1111 is *not* reported for the following code:

```csharp
[
Obsolete
]
```

I'm not sure what the expected behavior is, but I'm fairly sure it's one of the following:

1. SA1111 never should have been reported. If it was not reported for the original sample above, then it also would not have been confusing.
2. If SA1111 is supposed to be reported here, then both of the following changes need to be made:
  1. Use a different message which is less confusing.
  2. Update the code to still report the diagnostic for the second example.

**Comment by vweijsters** [comment]
The expected behavior should be that SA1111 is never reported. I verified that the old StyleCop also does not raise SA1111 in this case.

**Comment by sharwell** [comment]
@vweijsters What does it do in this case?

```csharp
[Conditional(""DEBUG""), Conditional(""TEST1"")
]
```

**Comment by vweijsters** [comment]
@sharwell No diagnostic is raised in that case.
